src/module.c

```diff
@@ -355,6 +355,8 @@ initpycurl(void)
         runtime_ssl_lib = "gnutls";
     } else if (!strncmp(vi->ssl_version, "NSS/", 4)) {
         runtime_ssl_lib = "nss";
+    } else if (!strncmp(vi->ssl_version, "mbedTLS/", 2)) {
```

Comment:
The second argument is the length of the string being compared, which should be 8 in this case.

```suggestion
    } else if (!strncmp(vi->ssl_version, "mbedTLS/", 8)) {
```